tests/torch/test_compile_keras.py

```diff
@@ -58,19 +25,11 @@ def compile_and_test_keras(
     input_shape = (input_output_feature,)
     n_examples = 50
 
-    # Define the Keras model
-    keras_model = model()
-    keras_model.build((None,) + input_shape)
-    keras_model.compute_output_shape(input_shape=(None, input_output_feature))
-
     # Create random input
     inputset = numpy.random.uniform(-100, 100, size=(n_examples, *input_shape))
 
     # Convert to ONNX
-    output_onnx_file_path = Path(tempfile.mkstemp(suffix=".onnx")[1])
```

Comment:
does it mean we don't test keras anymore? should we keep saying we are keras/tf friendly?

Reply:
we use pre-built onnx files so we still test for keras compat